pkg/grpc/server.go

```diff
@@ -218,6 +219,8 @@ func (s *server) getMiddlewareOptions() []grpc_go.ServerOption {
 		intr = append(intr, diag.DefaultGRPCMonitoring.UnaryServerInterceptor())
 	}
 
+	intr = append(intr, s.getGRPCAPILogging())
```

Comment:
If `api-log-level` is "off" or "", we don't even need to execute this line. Same for HTTP.